invenio.cfg

```diff
@@ -562,3 +572,92 @@ VOCABULARIES_NAMES_SCHEMES = {
 # We override the templates to add new fields needed for the migrated statistic events
 _APP_RDM_STATS_EVENTS["file-download"]["templates"] = "cds_rdm.stats.templates.events.file_download"
 _APP_RDM_STATS_EVENTS["record-view"]["templates"] = "cds_rdm.stats.templates.events.record_view"
```

Comment:
Also, take into account that we modify inline the `STATS_EVENTS` from invenio-app-rdm